### DISCUSSION TITLE
Table doesnt exist

### DISCUSSION THREAD

**Discussion by ThiagoUnder** [planetscale/discussion#491] [Q&A] [answered]
Hi, is there any docs on Drizzle connection. Ive generated the tables with drizzle-kit generate:mysql and set up connection with 
```
import { drizzle } from 'drizzle-orm/planetscale-serverless'
import { connect } from '@planetscale/database'

const connection = connect({
  host: process.env.DATABASE_HOST,
  password: process.env.DATABASE_PASSWORD,
  username: process.env.DATABASE_USERNAME,
})

export default drizzle(connection)
```
but Im getting error: DatabaseError: target: fakeep.-.primary: vttablet: rpc error: code = NotFound desc = Table 'fakeep.notes' doesn't exist (errno 1146) (sqlstate 42S02) (CallerID: gabf7y9fxht8uoa2me3q): Sql: "select id, content, completed, created_at, user_id from notes", BindVars: {REDACTED}. Db name is fakeep and table is notes. Im not sure if it's trying to connect to a table named "fakeep.notes".

If anyone cares to take a look:
https://github.com/ThiagoUnder/minimalpscale

Thank you!!

**Comment by jgreet** [comment, marked as answer]
I answered your support ticket but for anyone else stumbling on to this, that error is correctly saying the table `notes` doesn't currently exist in the database. `fakeep.notes`  is the same `notes` table in `<database-name>.<table-name>` format.

[drizzle-kit generate:mysql](https://orm.drizzle.team/kit-docs/commands) will create migration files for you, but you'll have to run those migrations against the database before using the new tables:

See https://orm.drizzle.team/kit-docs/overview#running-migrations

**Reply by raming** [reply]
After pushing my migration scripts to planetscale, I am still having a similar issue. I double check on the Planet Scale portal and my tables are created fine.
